.github/workflows/prod.yml

```
@@ -0,0 +1,93 @@
+        env:
+          PR_NUMBER: ${{ steps.get_pr_number.outputs.PR_NUMBER }}
+        run: |
+          DOCKER_IMAGE_TAG="1.1.$PR_NUMBER"
+          echo "Docker Image Tag: $DOCKER_IMAGE_TAG"
+          echo "DOCKER_IMAGE_TAG"="${DOCKER_IMAGE_TAG}" >> $GITHUB_OUTPUT
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v2
```

actionlint (1.7.4)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)